### PR TITLE
Use correct typographic symbols in guidelines

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR replaces typewriter apostrophes and quotation marks with the correct typographic ones and unifies all occurrences.

### CODE REVIEW

**Comment by craigsapp** [comment]
The apostrophes are fixed but single-quotation endings are made worse:

Screen Shot 2022-02-07 at 8 09 13 AM

(Starting quotes are being used for ending quotes).

**Comment by craigsapp** [comment]
Never mind: zooming in on them, I can now see that they are different...  I searched for one, but the web browser also matches to the other.

File: source/docs/03-metadata.xml [hunk 1]
```diff
@@ -1528,7 +1528,7 @@
...
...t>height</att>) of the parent element (<abbr>e.g.</abbr>, <gi scheme="MEI">folium</gi>) indicate the size of the bounding box of the remaining part of the page. That is, if the complete lower half of a page has been cut, the <att>width</att> and <att>height</att> attributes describe the remaining upper half. If, in contrast, only the lower right quarter of the page has been cut, these attributes still indicate the size of the full page (assuming that the removed section was a regular rectangle).The dimensions (<att>width</att>, <att>heigh</att>) on cutout itself are only to be used when there is a "gap" in the manuscript that allows to specify the dimensions of that missing part. In this case, the bounding box dimensions are given, together with <att>x</att> and <att>y</att> to indicate the upper left point on the original page. If, however, the removed section is available by itself, then a corresponding <gi scheme="MEI">folium</gi> (or <gi scheme="MEI">bifolium</gi>) should be placed inside the cutout element, and should provide it’s own dimensions using <att>width</att> and <att>height</att> there. In this case, <att>width</att> and <att>height</att> on cutout is expendable.</p>
```
Comment (bwbohl) [inline source/docs/03-metadata.xml]: can't spot the difference here…
shouldn't `"gap"` be converted to `“gap”`

File: source/docs/05-mensural.xml [hunk 2]
```diff
@@ -190,11 +190,11 @@
...
...i rend="italic">breves</hi>. This <hi rend="italic">longa</hi> can be "partially imperfected" by a following/preceding <hi rend="italic">semibrevis</hi>. This <hi rend="italic">semibrevis</hi> causes part of the <hi rend="italic">longa</hi>—one of its perfect <hi rend="italic">breves</hi>—to be imperfected, taking away one-third of one of its two halves. In this case, the <hi rend="italic">longa</hi>’s value changes from 6 <hi rend="italic">semibreves</hi> (two perfect <hi rend="italic">breves</hi>) into 5 <hi rend="italic">semibreves</hi>. Partial imperfection is not supported by the <att>dur.quality</att> attribute—because there can be many cases of partial imperfection, as will be seen in the following examples. To encode a partial imperfection, the <att>num</att> and <att>numbase</att> pair of attributes are used instead. Given the change in the <hi rend="italic">longa</hi>’s value from 6 <hi rend="italic">semibreves</hi> to 5 <hi rend="italic">semibreves</hi>, the corresponding attributes to encode this particular case of partial imperfection would be <hi rend="italic">num="6"</hi> and <hi rend="italic">numbase="5"</hi> as shown below in the code snippet and its rendering.</p>
```
Comment (bwbohl) [inline source/docs/05-mensural.xml]: "" --> “”? e.g. partially imperfect

File: source/docs/05-mensural.xml [hunk 3]
```diff
@@ -267,7 +267,7 @@
...
+                  <p>It is common in <hi rend="italic">Ars antiqua</hi> and some <hi rend="italic">Ars nova</hi> pieces to have no mensuration signs. In this case, the mensuration—the division levels corresponding to <hi rend="italic">modus maior</hi>, <hi rend="italic">modus minor</hi>, <hi rend="italic">tempus</hi>, and <hi rend="italic">prolatio</hi>—is given by the context. The next example shows the incipit of a four-voice piece, Josquin’s <hi rend="italic">Tu solus qui facis mirabilia</hi>, where only two of the voices (<hi rend="italic">Cantus</hi> and <hi rend="italic">Tenor</hi>) have a mensuration sign. The other two (<hi rend="italic">Altus</hi> and <hi rend="italic">Bassus</hi>) have <hi rend="italic">no mensuration signs</hi>, and the <hi rend="italic">mensura is given by the context</hi>. Therefore, while only the <hi rend="italic">Cantus</hi> and the <hi rend="italic">Tenor</hi> have attributes for encoding the mensuration sign (in this case, <att>mensur.sign</att> and <att>mensur.slash</att>), all four voices include attributes to encode the <hi rend="italic">mensura</hi> (<att>tempus</att> and <att>prolatio</att>). <figure>
```
Comment (bwbohl) [inline source/docs/05-mensural.xml:270]: difference?

File: source/docs/04-cmn.xml [hunk 4]
```diff
@@ -326,11 +326,11 @@
...
-                        <p>Some notational features like articulations or lyrics are connected to a whole chord instead of a single note. Therefore elements like <gi scheme="MEI">artic</gi> or <gi scheme="MEI">verse</gi> are also allowed as children of <gi scheme="MEI">chord</gi> elements. In the follwing example from Sergei Rachmaninoff's <hi rend="italic">Prelude in C-sharp minor</hi>, Op. 3, No. 2 all chords carry an accent.</p>
+                        <p>A chord is any set of pitches consisting of multiple notes that are to be played simultaneously and are usually grouped together visually with a single stem. In MEI the <gi scheme="MEI">chord</gi> element functions as a container for all participating notes. Also it features many attributes that are allowed for notes, e.g., usually all notes in a chord have a common duration, so it can be applied to the whole chord within it’s <att>dur</att> attribute.</p>
```
Comment (bwbohl) [inline source/docs/04-cmn.xml]: `<abbr>`got lost

File: source/docs/04-cmn.xml [hunk 5]
```diff
@@ -569,8 +569,8 @@
...
+                  <p>The <att>beam.group</att> attribute can be used to set a default beaming pattern to be used when no beaming is indicated at the layer level. It must contain a comma-separated list of time values that add up to a measure in the current meter, e.g., ‘4,4,4,4’ in 4/4 time indicates that each quarter note worth of shorter notes should be beamed together. Parentheses can be used to indicate sub-groupings of secondary beams. For example, ‘(4.,4.,4.)’ in 9/8 meter indicates one primary beam per measure with secondary beams broken at each dotted quarter duration, while ‘(4,4),(4,4)’ in 4/4 will result in a measure of 16th notes being rendered with a primary beam covering all the notes and secondary beams for each group of four 16th notes.</p>
```
Comment (bwbohl) [inline source/docs/04-cmn.xml]: `<abbr>`got lsot

File: source/docs/04-cmn.xml [hunk 6]
```diff
@@ -814,7 +814,7 @@
...
+                  <p>In some situations, a tuplet is made up of events in different measures. As this raises the issue of non-concurrent hierarchies, it is not possible to encode such situations with the <gi scheme="MEI">tuplet</gi> element described above. Therefore, MEI offers the <gi scheme="MEI">tupletSpan</gi> element, which is member of the <ident type="class">model.controlEventLike</ident> class. It is nested inside of <gi scheme="MEI">measure</gi>, following all the measure’s <gi scheme="MEI">staff</gi> children. It uses the same attributes as <gi scheme="MEI">tuplet</gi> to describe tuplets, but instead of nesting all affected notes inside itself, it references the <att>xml:id</att> values of all affected notes in its <att>plist</att> attribute and the initial and terminal notes of the tuplet using <att>startid</att> and <att>endid</att> attributes. This configuration allows tuplets to cross beams or measure boundaries. The following example demonstrates a typical example of such hierarchy-crossing tuplets:</p>
```
Comment (bwbohl) [inline source/docs/04-cmn.xml:817]: difference?

File: source/docs/04-cmn.xml [hunk 7]
```diff
@@ -1450,7 +1450,7 @@
...
+                  <p>This element allows the encoder to represent ornaments as textual strings (e.g., with a Unicode symbol) or with a user defined symbol. Chromatic auxiliaries can be represented with <att>accidlower</att> and <att>accidupper</att>. The <gi scheme="MEI">ornam</gi> element can also be a control element. That is, it can be linked via its attributes to other events. The starting point of the directive may be indicated by either a tstamp, tstamp.ges, tstamp.real or startid attribute, while the ending point may be recorded by either a tstamp2, dur, dur.ges or endid attribute. It is a semantic error not to specify a starting point attribute.</p>
```
Comment (bwbohl) [inline source/docs/04-cmn.xml]: `<abbr>`got lost

**Review by bwbohl — COMMENTED** [review]
generally I like the idea of pimping up typography in the guidelines. In some casese I wonder though, whether it would be better to come up with a stronger typed markup, e.g. for attribute values that could be markedUp using `<tei:val>`

File: source/docs/05-mensural.xml (hunk 3 above)
Comment (rettinghaus) [reply source/docs/05-mensural.xml:270]: apostrophe in Josquin’s

File: source/docs/04-cmn.xml (hunk 4 above)
Comment (rettinghaus) [reply source/docs/04-cmn.xml]: There were some more, thanks!

File: source/docs/04-cmn.xml (hunk 6 above)
Comment (rettinghaus) [reply source/docs/04-cmn.xml:817]: apostrophe in measure’s

**Comment by rettinghaus** [comment]
@bwbohl I like the idea using `val` but that should probably go with another PR. We need to add a matching rule to the css.

**Review by bwbohl — APPROVED** [review]

**Comment by kepper** [comment]
as per consensus at ODD meeting Feb 2022. Many thanks :)

**Pull request merged**